ush/ioda/bufr2ioda/marine/b2i/ocean.py

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env python3
+
+# import sys
+import numpy as np
+import numpy.ma as ma
+# import os
+# import argparse
+import math
+# import calendar
+# import time
+# import copy
+# from datetime import datetime
+# import json
+# from pyiodaconv import bufr
+# from collections import namedtuple
+# from pyioda import ioda_obs_space as ioda_ospace
+# from wxflow import Logger
+# import warnings
+# # suppress warnings
+# warnings.filterwarnings('ignore')
+# import subprocess
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import netCDF4 as nc
+import xarray as xr
+
+
+
+ocean_basin_nc_path = "/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/static/common/RECCAP2_region_masks_all_v20221025.nc"
```

Comment:
That only works on hera. 
The RECCAP2_region_masks_all_v20221025.nc file is staged before your converters are used, so you could assume that it is going to be present at the same location the converters are running. Add some logic to trap the error if the file doesn't exist and send a warning instead of aborting or letting the code fail.